wishlist-backend/wishlist/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+# Create your models here
+
+import uuid
+
+from users.models import User
+from products.models import Product
+
+class Wishlist(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    products = models.ManyToManyField(Product, related_name='products_list')
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Why do you even need a foreign key? is it for indexing?

Reply:
I have kept the userid as foriegn key in Wishlist for maintaining referential integrity by ensuring that there are no orphaned rows in the wishlist table. So at anytime, if a user is deleted, its wishlists enteries in the db will get deleted. Does that makes sense?
Never thought it in the prospect of indexing.

Reply:
Makes sense.